Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -523,6 +528,9 @@ class BrowserViewController: UIViewController {
 
     override func viewWillDisappear(animated: Bool) {
         screenshotHelper.viewIsVisible = false
+        
+        log.debug("BVC stopping handoff.")
+        HandoffManager.sharedInstance.stop()
```

Comment:
I noticed that in all of the other calls to stop we also clear the URL but not here. How come? Would it make sense to clear every time we stop?